src/D2L.Security.OAuth2/Provisioning/Default/AuthServiceClient.cs

```diff
@@ -60,7 +60,7 @@ Uri authEndpoint
 			}
 
 			m_client = httpClient;
-			m_tokenProvisioningEndpoint = new Uri( authEndpoint + TOKEN_PATH );
+			m_tokenProvisioningEndpoint = authEndpoint.RelativePathAsNonLeaf( TOKEN_PATH );

```

Comment:
This would result in `/foo//connect/token` depending on the input authEndpoint.

This was addressed before in-product by removing the trailing slash from the input, but looking to handle it regardless here.